catroid/src/main/AndroidManifest.xml

```diff
@@ -198,6 +198,7 @@
             android:noHistory="false"
 
             android:screenOrientation="portrait" />
+        <activity android:name=".ui.MultilingualActivity" />
```

Comment:
can you just remove this. then i can merge this!

Reply:
done &PR now has one commit.